docs/supported.md

```diff
@@ -254,7 +286,7 @@ Macros accept up to ten arguments: #1, #2, etc.
 
 Available functions include:
 
-`\mathchoice` `\TextOrMath` `\@ifstar` `\@ifnextchar` `\@firstoftwo` `\@secondoftwo` `\relax`
+`\mathchoice` `\TextOrMath` `\@ifstar` `\@ifnextchar` `\@firstoftwo` `\@secondoftwo` `\relax` `\allowbreak` `\nobreak`
```

Comment:
I'm not sure this is the right place for the `break` commands.  Maybe in "Line Breaks and Vertical Layout" around `\\` and `\newline`?

Reply:
`\char` is another recent addition for this list.